src/configure/configure.ts

```diff
@@ -513,7 +513,7 @@ class Orchestrator {
         }
 
         if (extensionVariables.templateServiceEnabled) {
-            telemetryHelper.setTelemetry(TelemetryKeys.ChosenTemplate, this.inputs.pipelineConfiguration.templateInfo.label);
+            telemetryHelper.setTelemetry(TelemetryKeys.ChosenTemplate, this.inputs.pipelineConfiguration.templateInfo.templateDescription);
```

Comment:
this might cause an issue if we want to look at both current and historical data. the format is changing.
Should we rather log `<label>:<targetKind>` ?

Reply:
This seems to be pretty old code as ``` pipelineConfiguration.templateInfo ``` doesn't exist anymore.

Reply:
yes it doesn't exist, this branch was not updated after your changes.